src/components/CheckboxGroup/CheckboxGroup.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { useEffect, useId, useReducer } from 'react';
+import cn from 'classnames';
+
+import { Checkbox, ErrorMessage } from '@/components';
+
+import classes from './CheckboxGroup.module.css';
+
+export interface CheckboxItem {
+  checked?: boolean;
+  description?: string;
+  disabled?: boolean;
+  id?: string;
+  label?: string;
+  name: string;
+}
```

Comment:
Should we perhaps [use `Pick`](https://www.typescriptlang.org/docs/handbook/utility-types.html) to ensure the common parts of the interface between the `Checkbox` component and this checkbox list item are in sync?